### PR TITLE
Cleanup ssh module

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: puppet/modules/ssh/manifests/init.pp [hunk 1]
```diff
@@ -1,15 +1,4 @@
 class ssh {
-  file { '/root/.ssh':
-    ensure => directory,
-    owner  => 'root',
-    group  => 'root',
-  }
-
-  file { '/root/.ssh/authorized_keys':
-    ensure  => file,
-    content => $::root_ssh_recovery_key,
```
Comment (mmoll) [inline puppet/modules/ssh/manifests/init.pp:10]: This was not set for at least some years.

File: puppet/modules/ssh/manifests/init.pp [hunk 2]
```diff
@@ -1,15 +1,4 @@
 class ssh {
-  file { '/root/.ssh':
-    ensure => directory,
-    owner  => 'root',
-    group  => 'root',
-  }
-
-  file { '/root/.ssh/authorized_keys':
-    ensure  => file,
```
Comment (ekohl) [inline puppet/modules/ssh/manifests/init.pp:9]: Should we ensure it's absent?

Comment (mmoll) [reply puppet/modules/ssh/manifests/init.pp:9]: Shouldn't make a difference with `PermitRootLogin no` ;)

Comment (mmoll) [reply puppet/modules/ssh/manifests/init.pp:9]: changed anyway

**Pull request merged**